docs/DeveloperGuide.md

```diff
@@ -48,6 +48,17 @@ API: `Storage.java`
 * The `load` method in LoadData reads the `data.txt` file and loads any existing Income, Expense and Budget into the application.
 * The `save` method in SaveData saves all Incomes, Expenses and existing Budget into the `data.txt` file.
 
+#### Design considerations:
+
+* There are 2 main ways to implement the storage, one is to save the data after every command, and the other is to save 
+the data one upon exiting the program with the `exit` command.
+* Saving the data once upon exit (Currently implemented):
+  * Advantage: Better efficiency and performance of the program.
+  * Disadvantage: If the program crashes or is exit incorrectly, data will not be saved.
```

Comment:
"exited" !!!!!!!!!
or "crashes or exits" !!!!

Reply:
💀 i changed it to the second one